tests/test_climate_baseline_helpers.py

```diff
@@ -57,23 +49,23 @@ def test_creates_the_expected_new_network_record(empty_database_session):
         assert result.publish == True
         assert 'PCIC' in result.long_name
 
-    def test_creates_no_more_than_one_of_them(empty_database_session):
-        sesh = empty_database_session
+    def it_creates_no_more_than_one_of_them(session):
+        sesh = session
```

Comment:
Again, the aliasing is probably not necessary in this case. Doesn't save that much space.